Microsoft.Toolkit.Uwp.UI/Converters/DateTimeToStringConverter.cs

```diff
@@ -0,0 +1,62 @@
+﻿// ******************************************************************
```

Comment:
But the ViewModel is supposed to be where what controls how the data is presented to the view.